console/console-init/ui/src/modules/address-space/components/AddressSpaceReview/AddressSpaceReview.tsx

```diff
@@ -3,7 +3,8 @@
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 
-import * as React from "react";
```

Comment:
same here. Any specific reason to just import React instead of * as React statment?

Reply:
Difference between *as and normal import is * as import includes all the modules and normal import includes default imports only. At this point we need only default imports not whole modules.